google_maps/google_maps.py

```diff
@@ -2,11 +2,13 @@
 
 import pkg_resources
 from django.utils.translation import ugettext_lazy as _
+import urllib
 
 from xblock.core import XBlock
-from xblock.fields import Scope, Integer, String
+from xblock.fields import Dict, Integer, Scope, String
 from xblock.fragment import Fragment
 from xblockutils.resources import ResourceLoader
+from xmodule.services import SettingsService
```

Comment:
This is part of the edx-platform, which is not a requirement for our XBlock. Don't go breaking the 4th wall.